### ISSUE TITLE
share_survey does not allow to remove survey from users' account

### ISSUE THREAD

**Issue by ppssphysics** [ppssphysics/QualtricsInR#3]
When setting all sharing permissions to FALSE, one would expect to see the survey removed from the user's account with whom the survey was initially shared. Right now this is not the case and the survey stays there with only a preview option available.

**Comment by jmobrien** [comment]
Just FYI--I took this issue up with Qualtrics myself a while back, and it does not appear actual collaborator lists can be managed through the api, only permissions. 

I also don't think you can check who is on any given survey, so confirmation is challenging, and if someone was put on through the web interface and you don't know it you could have a problem as well.  That information could be mistaken or outdated, however.

Neat package, btw.

**Comment by ppssphysics** [comment]
@jmobrien absolutely. My last exchange with Qualtrics a few months back indicated that was still not possible, which is quite unfortunate. At least setting all permission to FALSE prevents any kind of interaction with the survey. But even simple preview permission is potentially an issue in certain circumstances.

I am keeping this issue open to remind myself to check in with Qualtrics from time to time. You probably noticed for yourself their API documentation is not the easiest to navigate.

**Comment by jmobrien** [comment]
Yes, I absolutely have noticed that about their documentation.  If you haven't been over there in a while, there was a semi-recent redesign over at api.qualtrics.com that has marginally improved what was documented, especially regarding what returns from different endpoints.  It's not an unqualified improvement in clarity, though.  

Full agree on everything you say about API collaborator management.  Beginning this year I took on oversight of over 200+ surveys where it wasn't fully clear or consistent who was even shared on each one.  Without at least that info the API is useless for access management.  I don't even fully know what user ID's I would need to run for each survey--and even if I just found them all and ran a batch command blanking permissions, I'm believe that would end up actually ADDING people as collaborators to surveys they weren't previously on.  IMO something that Qualtrics should be way more clear about, maybe relevant here if you're accessing that endpoint.

It's legacy, though, right?  Maybe they just don't care.  What gets me is that I don't think brand admins have good tools for this either.  They can easily change who owns a survey, even in batches, but my admins suggested they couldn't manage collaborators as a survey owner might.  Manual review of 200+ permissions is unsafe and no fun.
